docs/content/configuration/prefill-disk-cache.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Pre-filling the Disk Cache
+description: How to pre-fill the disk cache
+weight: 4
+---
+
+One of the popular features of Athens is that it can be run completely cut off to the internet. In this case, though, it can't reach out to an upstream (e.g. a VCS or another module proxy) to fetch modules that it doesn't have in storage. So, we need to manually fill up the disk partition that Athens uses with the dependencies that we need.
+
+This document will guide you through packaging up a single module called `github.com/my/module`, and inserting it into the Athens disk storage.
+
+# First, get the tools
+
+You'll need to produce the following assets from module source code:
+
+- `source.zip` - just the Go source code, packaged in a zip file
+- `go.mod` - just the `go.mod` file from the module
+- `$VERSION.info` - metadata about the module
+
+The `source.zip` file has a specific directory structure and the `$VERSION.info` has a JSON structure, both of which you'll need to get right in order for Athens to serve up the right dependency formats that the Go toolchain will accept.
+
+>We don't recommend that you create these assets yourself. Instead, use [pacmod](https://github.com/plexsystems/pacmod)
+
+To install the `pacmod` tool, run `go install` like this:
+
+```console
+$ go install github.com/plexsystems/pacmod
```

Comment:
I think we should use `go get` here. `go install` assumes the code is already on the end-users machine.